compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5622,11 +5631,12 @@ static TR::Register *inlineArrayCmp(TR::Node *node, TR::CodeGenerator *cg)
 
 TR::Register *OMR::Power::TreeEvaluator::arraycmpEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   TR::Compilation *comp = cg->comp();
-   static char *disableP10ArrayCmp = feGetEnv("TR_DisableP10ArrayCmp");
-   if (cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P10) && !disableP10ArrayCmp)
-      return inlineArrayCmpP10(node, cg);
-   return inlineArrayCmp(node, cg);
```

Comment:
I think you accidentally removed the Power 10 implementation of arraycmp here.

Reply:
I moved the if statement calling the Power 10 implementation to the [beginning of `inlineArrayCmp`](https://github.com/eclipse/omr/blob/104d1f60f5a242078a88364fe5b78f54ef544db2/compiler/p/codegen/OMRTreeEvaluator.cpp#L5446-L5448) so it wouldn't need to be duplicated between [`arraycmpEvaluator`](https://github.com/eclipse/omr/blob/104d1f60f5a242078a88364fe5b78f54ef544db2/compiler/p/codegen/OMRTreeEvaluator.cpp#L5632) and [`arraycmplenEvaluator`](https://github.com/eclipse/omr/blob/104d1f60f5a242078a88364fe5b78f54ef544db2/compiler/p/codegen/OMRTreeEvaluator.cpp#L5637)